installer/PowerToysSetup/Product.wxs

```diff
@@ -635,6 +635,7 @@
         <!-- File to include common library used by preview handlers -->
         <File Source="$(var.BinX64Dir)modules\FileExplorerPreview\PreviewHandlerCommon.dll" />
         <File Source="$(var.BinX64Dir)modules\FileExplorerPreview\Telemetry.dll" />
+        <File Id="FileExplorerPreview_PowerToysInterop" Source="$(var.BinX64Dir)modules\FileExplorerPreview\PowerToysInterop.dll" />
```

Comment:
I assume this (+ interop project reference) needs to be added for every module that is using EventBase because they will start crashing when they will use it?

Reply:
On the long term, it may be better to not require the dependency on an interop dll for all C# projects, and provide a simplified way of getting the version string from https://github.com/microsoft/PowerToys/blob/master/installer/Version.props at compile time (since it's not necessary to load it at run-time).

Reply:
The props file is already included in the project so I don't need to use the interop.  The only advantage of using the interop was that it keeps the formatting consistent, however that's not worth the risk of having people potentially forget to update a wxs file on a new project that includes telemetry.